app/src/main/java/fr/free/nrw/commons/AboutActivity.java

```diff
@@ -33,4 +36,29 @@ public void onCreate(Bundle savedInstanceState) {
         versionText.setText(BuildConfig.VERSION_NAME);
         initDrawer();
     }
+
+    public void getOpenFacebookIntent(View view) {
```

Comment:
Strictly speaking, this isn't really a "get" method in the traditional Java sense as it's not returning anything.  Perhaps a name like "launchFacebook()" would be better?